src/util/text.ts

```diff
@@ -378,13 +379,11 @@ function lookupInUnicodeMap(code: u16, map: u16[]): bool {
 }
 
 function isUnicodeIdentifierStart(code: i32): bool {
-  return code < 170 || code > 65500 ? false :
-         lookupInUnicodeMap(code as u16, unicodeIdentifierStart);
+  return lookupInUnicodeMap(code as u16, unicodeIdentifierStart);
```

Comment:
The `isUnicodeIdentifierStart` and `isUnicodeIdentifierPart` functions seem to be either unnecessary or prone to error now, depending on perspective

Reply:
Yeah. Also think about to remove them